Masquerade4997Library/src/main/java/Library4997/MasqDriveTrains/MasqMechanumDriveTrain.java

```diff
@@ -96,11 +96,6 @@ public void setPowerMECH(double angle, double speed, double targetHeading) {
         rightDrive.motor2.setPower(rightBack);
     }
 
-
-    public void setTurnKP(double kp) {
-        turnController.setKp(kp);
-    }
-
```

Comment:
Why?

Reply:
I replaced the only 2 instances of it with turnController.setKp(kp);